package.json

```diff
@@ -216,7 +216,7 @@
 		"build-client": "npm run build-client-evergreen",
 		"build-client-fallback": "cross-env-shell BROWSERSLIST_ENV=defaults NODE_PATH=$NODE_PATH:server:client:. node $NODE_ARGS ./node_modules/webpack/bin/webpack.js --display errors-only",
 		"build-client-evergreen": "cross-env-shell BROWSERSLIST_ENV=evergreen NODE_PATH=$NODE_PATH:server:client:. node $NODE_ARGS ./node_modules/webpack/bin/webpack.js --display errors-only",
-		"build-client-both": "npm run build-client-fallback && npm run build-client-evergreen",
+		"build-client-both": "concurrently -c cyan -n \"fallback ,evergreen\" \"npm run build-client-fallback\" \"npm run build-client-evergreen\"",
```

Comment:
typo: you probably didn't intend to put a space before the comma? 🙂

Reply:
Good eye, but that space is actually intentional! It ensures that both labels have the same length (with `fallback` having an extra space at the end), and thus the output gets aligned:

```
[evergreen] Disabling server-side user-bootstrapping because of a missing secrets.json
[fallback ] Disabling server-side user-bootstrapping because of a missing secrets.json
[evergreen] (node:51995) DeprecationWarning: Tapable.plugin is deprecated. Use new API on `.hooks` instead
[fallback ] (node:51994) DeprecationWarning: Tapable.plugin is deprecated. Use new API on `.hooks` instead
```

It's silly, but I think it looks better 😁